### PR TITLE
Requirement for 0.0.2

### PR DESCRIPTION
**Pull request by trungdq88**
Please review requirement for 0.0.2 and put questions if any.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,47 @@ Name | Age
...
+<DataTable
+  data={[
+    { name: 'Trung', age: 123 },
+    { name: 'Quang' },
+  ]}
+  fields={[
+    ['Name', 'name'],
+    ['Age', 'age'],
+  ]}
+  style={{ border: '1px solid #ccc' }}
```
Comment (phongdoduy) [inline README.md]: Is it the style for the table ? 

For the `v0.0.2`:
Do we have the style for header of table ? of styles for every cells ?

File: README.md [hunk 2]
```diff
@@ -25,3 +25,47 @@ Name | Age
...
+<DataTable
+  data={[
+    { name: 'Trung', age: 123 },
+    { name: 'Quang' },
+  ]}
+  fields={[
+    ['Name', 'name'],
+    ['Age', 'age'],
+  ]}
+  emptyText="--"
```
Comment (phongdoduy) [inline README.md]: Should we use another name like `defaultCellValue` ? To show when there is no value for the cell 😃

File: README.md (hunk 1 above)
Comment (trungdq88) [reply README.md]: It is style for `<table>`. Style for header / row / cell could be added later.

File: README.md (hunk 2 above)
Comment (trungdq88) [reply README.md]: `defaultCellValue` is the same as `emptyText`, but let's use `emptyText` because it shorter.

Comment (phongdoduy) [reply README.md]: 🤔 OK, but I think we should not restrict the empty value as `text`, push it as `emptyValue`.

Comment (trungdq88) [reply README.md]: Every value will eventually get rendered as text in html. So we should only support 2 levels here:
- `emptyText`: return the text when property is `null` or `undefined`.
- `emptyRenderer(func)`: receive the property and return anything that React can render (string, number, element or React component)

Comment (phongdoduy) [reply README.md]: Yes, agreed 💯

**Pull request merged**